crates/router/src/core/payments/flows/setup_mandate_flow.rs

```diff
@@ -86,14 +87,87 @@ impl Feature<api::SetupMandate, types::SetupMandateRequestData> for types::Setup
         ))
         .await?;
 
-        mandate::mandate_procedure(
-            state,
-            resp,
-            maybe_customer,
-            pm_id,
-            connector.merchant_connector_id.clone(),
-        )
-        .await
+        if let Some(mandate_id) =
+            self.request
```

Comment:
Add a task to do validation for invalid fields like amount, currency in update mandate call

Reply:
As per our discussion , we would not be adding the validation anymore